VirtoCommerce.Platform.Web/Scripts/common/directives/dateTime-picker.js

```diff
@@ -395,9 +395,9 @@ angular.module('platformWebApp')
                     }
 
                     var newDate = scope.date ? dateFilter(scope.date, dateFormat, ngModelOptions.timezone) : null;
-
+                    
```

Comment:
Fix your formatting settings

Reply:
Could you give me formatting settings file for VS 2017 that i could import? Or what settings should i change?